scikitplot/decomposition.py

```diff
@@ -156,14 +166,36 @@ def plot_pca_2d_projection(clf, X, y, title='PCA 2-D Projection', ax=None,
     classes = np.unique(np.array(y))
 
     colors = plt.cm.get_cmap(cmap)(np.linspace(0, 1, len(classes)))
-
+    
+    vectors=np.transpose(clf.components_[0:2, :])
+    
+    xs = transformed_X[:,0]
+    ys = transformed_X[:,1]   
```

Comment:
Style: space after comma

Reply:
`vectors`, `xs`, and `ys` should be evaluated within the `if biplot` block since they're only used there.